plugin.video.redbull.tv/default.py

```diff
@@ -0,0 +1,82 @@
+import sys, urlparse, os, urllib, datetime, time
+import xbmcgui, xbmcplugin, xbmcaddon, xbmc
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'resources'))
```

Comment:
I'm not sure setting a system path is appropriate. Could you directly import your modules example. `from resources.lib import utils`  btw you may need init your folders, thx